OutOfSchool/OutOfSchool.DataAccess/OutOfSchool.DataAccess.csproj

```diff
@@ -28,12 +28,12 @@
       <PackageReference Include="H3Lib" Version="3.7.2" />
       <PackageReference Include="Microsoft.AspNetCore.DataProtection.EntityFrameworkCore" Version="6.0.6" />
       <PackageReference Include="Microsoft.AspNetCore.Identity.EntityFrameworkCore" Version="6.0.6" />
-      <PackageReference Include="Microsoft.EntityFrameworkCore" Version="6.0.6" />
-      <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="6.0.6">
+      <PackageReference Include="Microsoft.EntityFrameworkCore" Version="7.0.2" />
```

Comment:
Have you read:
[https://learn.microsoft.com/en-us/ef/core/what-is-new/ef-core-7.0/breaking-changes](https://learn.microsoft.com/en-us/ef/core/what-is-new/ef-core-7.0/breaking-changes)
It describes a few high impact changes.

Reply:
Thx. I don't know how to change this PR to draft after started it to review )))

Reply:
I am reading the article and i'll make some changes. Thx again.